ReactWindows/ReactNative/Views/TextInput/ReactTextInputManager.cs

```diff
@@ -489,14 +462,28 @@ public void SetSelectTextOnFocus(ReactTextBox view, bool selectTextOnFocus)
         }
 
         /// <summary>
-        /// Sets the max height of the text box.
+        /// Sets whether the view is a tab stop.
         /// </summary>
         /// <param name="view">The view instance.</param>
-        /// <param name="height">The max height.</param>
-        [ReactProp("maxHeight")]
-        public void SetMaxHeight(ReactTextBox view, double height)
```

Comment:
>SetMaxHeight [](start = 20, length = 12)

Why are you deleting `SetMaxHeight`?

Reply:
A merging mistake, I'll be adding back.
Yet @Eric, isn't this property "shadowed" by the similar one defined in the associated shadow node? It seems those handlers still have overriding behavior.

---
In reply to: [157095896](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r157095896) [](ancestors = 157095896)